CI seems to fail after yesterdays iris v3.1.0 release

Output CI:

```bash
Run pytest --cov=pyaerocom/
ImportError while loading conftest '/home/runner/work/pyaerocom/pyaerocom/tests/conftest.py'.
tests/conftest.py:16: in <module>
    import pyaerocom.testdata_access as td
pyaerocom/__init__.py:13: in <module>
    from . import io
pyaerocom/io/__init__.py:76: in <module>
    from .readungriddedbase import ReadUngriddedBase
pyaerocom/io/readungriddedbase.py:13: in <module>
    from pyaerocom.helpers import varlist_aerocom
pyaerocom/helpers.py:33: in <module>
    STR_TO_IRIS = dict(count       = iris.analysis.COUNT,
E   AttributeError: module 'iris' has no attribute 'analysis'
Error: Process completed with exit code 4.
```

--- a/pyaerocom_env.yml
+++ b/pyaerocom_env.yml
@@ -3,7 +3,7 @@
 channels:
   - conda-forge
 dependencies:
-  - iris >=3.0.1
+  - iris >=3.0.1,<3.1.0
   - xarray >=0.16.0
   - cartopy >=0.16.0
   - matplotlib >=3.0.1

--- a/setup.py
+++ b/setup.py
@@ -52,7 +52,7 @@ setup(
 
     python_requires = '>=3.7,<4',
     install_requires = [
-        'scitools-iris>=2.1.0',
+        'scitools-iris>=3.0.1,<3.1.0',
         'scitools-pyke>=1.1.1',
         'xarray>=0.16.0',
         'cartopy>=0.16.0',

